Add option to disable XML RPC (#109)
humanmade/altis-cms: docs/xmlrpc.md

# XML RPC

> WordPress XML RPC is a remote procedure call which uses XML to encode its calls and HTTP as a transport mechanism.

Reviewer: ```suggestion 
XML RPC (Remote Procedure Call) is a type of API which uses XML to encode its calls and HTTP as a transport mechanism.
```

> It is used by certain applications to enable remote management of WordPress sites, for example certain mobile apps for moderating comments, or the Jetpack plugin.

Reviewer: ```suggestion
It is used to enable remote management of applications, for example via mobile or desktop apps.
```
Let's not mention Jetpack :)

> By default the XML RPC is enabled. If you need to disable it, set the `xmlrpc` configuration property to `false`

Reviewer: ```suggestion
By default XML RPC is enabled. If you wish to disable it, set the `xmlrpc` configuration property to `false`
```
